src/public/sql/data.sql

```diff
@@ -0,0 +1,39 @@
+
+INSERT IGNORE INTO users (id, name, first_name, last_name, email, password)
+ VALUES
+ (1, 'admin', 'admin-firstname', 'admin-lastname', 'admin@upei.ca', '1234qwer'),
+ (2, 'st1', 'st1-firstname', 'st1-lastname', 'st1@upei.ca', '1234qwer'),
+ (3, 'st2', 'st2-firstname', 'st2-lastname', 'st2@upei.ca', '1234qwer'),
+ (4, 'ept1', 'ept1-firstname', 'ept1-lastname', 'ept1@upei.ca', '1234qwer');
+
+INSERT IGNORE INTO roles (id, name, slug)
+ VALUES
+    (1, 'administrator', 'administrator'),
```

Comment:
We need to include TA in here, they should have the ability to do everything except delete (which hasn't even been made yet)